src/charts/iot-edge-connector/values.yaml

```diff
@@ -7,14 +7,23 @@ edgeproviderimage:
   secretKey: hub0-cs
 vkimage:
   repository: microsoft/virtual-kubelet
-  tag: 0.3
+  tag: latest
   pullPolicy: Always
 env:
   nodeName: iot-edge-connector-hub0
-  nodeTaint:
+  nodeOsType: Linux
+  monitoredNamespace: ""
+  # 'provider' must be 'web'
+  provider: web
+taint:
+  enabled: true
+  key: virtual-kubelet.io/provider
+  value: iotedge
+  # 'effect' must be 'NoSchedule`, 'PreferNoSchedule' or 'NoExecute'.
```

Comment:
I suppose in our case we _really_ want this to be `NoSchedule` :)

Reply:
Doesn't matter. NoSchedule is the preferred one, but the other one's also work. So, I would leave as it is right now with the comment.

Reply:
Doesn't `PreferNoSchedule` mean that the controller will only _try_ not to schedule pods on this node? So there's a possibility that it might still happen?